### PR TITLE
change Notify when to Notify once

### PR DESCRIPTION
**Pull request by bituka**
My wife told me that it is better to use once instead when in one of the bullet points

### CODE REVIEW

**Comment by bituka** [comment]
As the other steps have no "s", this needs to change for uniformity.

File: README.md [hunk 1]
```diff
@@ -37,8 +37,8 @@
 * Concatenate And Minify Scripts
 * Analyze JavaScript with ESLint
 * Compile ES2015
-* Notify when tasks are complete
-* Monitors change in the files and reload browser with BrowserSync
+* Notify once tasks are complete
```
Comment (tiagoporto) [inline README.md:40]: The sentence now is Notifies when tasks are complete.
Can you confirm the `once` use please.

**Review by tiagoporto — CHANGES_REQUESTED** [review]
There some changes now, please resolve the conflicts to merge.
Thanks

**Comment by bituka** [comment]
I think this seems fine now.

**Pull request closed without merging**